stable/burrow/README.md

```diff
@@ -1,164 +1,183 @@
-# burrow
+# Burrow
 
-[burrow](https://github.com/hyperledger/burrow) is a permissioned Ethereum smart-contract blockchain node. It executes Ethereum smart contract code on a permissioned virtual machine. Burrow provides transaction finality and high transaction throughput on a proof-of-stake Tendermint consensus engine.
+[Burrow](https://github.com/hyperledger/burrow) is a permissioned Ethereum smart-contract blockchain node which provides transaction finality and high transaction throughput on a proof-of-stake [Tendermint](https://tendermint.com) consensus engine.
 
 ## TL;DR;
 
-```console
-$ helm install stable/burrow
+```bash
+helm install stable/burrow
 ```
 
 ## Introduction
 
 This chart bootstraps a burrow network on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
-## Installing the Chart
+## Installation
 
-To install the chart with the release name `my-release`:
+### Prerequisites
 
-```console
-$ helm install stable/burrow --name my-release
+To deploy a new blockchain network, this chart requires that two objects be present in the same Kubernetes namespace: a configmap should house the genesis file and a secret should hold any validator keys. The provided script, `initialize.sh` automatically provisions a number of files using the [burrow](https://github.com/hyperledger/burrow) toolkit, so please first ensure that `burrow --version` matches the `image.tag` in the [configuration](#configuration). This sequence also requires the very fine [jq](https://stedolan.github.io/jq/) binary to be installed. Two files will be generated, the first of note is `chain-info.yaml` which contains the two necessary Kubernetes specifications to be added to the cluster:
```

Comment:
Let's make it clearer in the second sentence that we need `burrow` and `jq` installed on the operator's environment in order for them to spin up the network.